### PR TITLE
Add support for redis command: CLIENT TRACKINGINFO

### PR DESCRIPTION
**Pull request by cemasma**
This pull request covers the implementation of the issue below to support CLIENT TRACKINGINFO command.
https://github.com/redis/jedis/issues/2881

### CODE REVIEW

File: src/main/java/redis/clients/jedis/BuilderFactory.java [hunk 1]
```diff
@@ -2004,6 +2004,19 @@ public String toString() {
     }
   };
 
+  public static final Builder<TrackingInfo> TRACKING_INFO_RESPONSE = new Builder<TrackingInfo>() {
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/BuilderFactory.java:2007]: For this class, a builder in either `TrackingInfo` or here (`BuilderFactory`) is enough. Two builders in both places is unnecessary.

**Review by sazzad16 — CHANGES_REQUESTED** [review]
Thank you for the PR. Please:
1. Cover RESP3 Reply.
2. Test RESP3 Reply.

File: src/main/java/redis/clients/jedis/BuilderFactory.java (hunk 1 above)
Comment (cemasma) [reply src/main/java/redis/clients/jedis/BuilderFactory.java]: Hey @sazzad16 thanks for the review! I have updated the pr accordingly 👍

File: src/main/java/redis/clients/jedis/resps/TrackingInfo.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    public List<String> getPrefixes() {
+        return prefixes;
+    }
+
+    public static final Builder<TrackingInfo> TRACKING_INFO_BUILDER = new Builder<TrackingInfo>() {
+        @Override
+        public TrackingInfo build(Object data) {
+            List commandData = (List) data;
+
+            if (commandData.get(0) instanceof KeyValue) {
```
Comment (cemasma) [inline src/main/java/redis/clients/jedis/resps/TrackingInfo.java:41]: https://github.com/redis/jedis/pull/3751#pullrequestreview-1926556648
This one has also been completed btw with its test [here](https://github.com/redis/jedis/pull/3751/commits/bac0296e5f294a34f1dadb5d86b856278674c38c#diff-5482ee92f736fc02b8cad49e22343bfabffaf25fc3a32a3ba512ee1c979e22b6R295)

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**